vertical-pod-autoscaler/pkg/admission-controller/main.go

```diff
@@ -75,6 +79,8 @@ func main() {
 		Addr:      ":8000",
 		TLSConfig: configTLS(clientset, certs.serverCert, certs.serverKey),
 	}
-	go selfRegistration(clientset, certs.caCert, &namespace)
+	Path := webhookConfigName + ":"
+	url := fmt.Sprintf(Path + *serverPort)
```

Comment:
These two lines can be shortened to url := fmt.Sprintf("%s:%v", webhookAddress, serverPort)